conf/application.conf

```diff
@@ -313,6 +313,11 @@ urls {
     wrong-credentials = "/business-account/wrong-credentials"
   }
 
+  tar{
+    host = "http://localhost:9280"
+    url = "/account"
+  }
```

Comment:
This should go inside the services section. app-config-base will need to be updated to match